script.js

```diff
@@ -287,6 +288,21 @@ function modalPopUp2() {
       nameLabel.innerText == completedMessages.nameCompleted &&
       emailLabel.innerText == completedMessages.emailCompleted
     ) {
+      //POST Request
+      const bookingResult = await fetch('https://lernia-sjj-assignments.vercel.app/api/booking/reservations', {
+      method: 'POST',
+      headers: {'Content-Type': 'application/json',},
+      body: JSON.stringify({
+        challenge: 3,
+        name: input1.value,
+        email: input2.value,
+        date: inputDate.value,
+        time: input3.value,
+        participants: parseInt(input4.value.match(/\d+/g)),
+    }),
+});
+const booking = await bookingResult.json();
+console.log(booking);
```

Comment:
Indentation is off here. It's the sort of thing that should ideally be caught in review, if not automated.